tests/resource-edit-managed-title-test.js

```diff
@@ -38,6 +38,37 @@ describeApplication('ResourceEditManagedTitleInManagedPackage', () => {
     });
   });
 
+  describe('visting the resource edit page but the resource is unselected', () => {
+    beforeEach(function () {
+      return this.visit(`/eholdings/resources/${resource.titleId}/edit`, () => {
+        expect(ResourceEditPage.$root).to.exist;
+      });
+    });
+
+    describe('with the resource unselected', () => {
+      beforeEach(() => {
+        return ResourceEditPage.toggleIsSelected();
+      });
+
+      it('should not display the coverage button', () => {
+        expect(ResourceEditPage.hasAddCustomCoverageButton).to.be.false;
+      });
+
+      it('displays a message to add title to holdings', () => {
+        expect('');
```

Comment:
Wasn't sure on the intent for this

Reply:
We were going to add a tests there then changed our mind.